guidelines.md

```diff
@@ -0,0 +1,62 @@
+## Programming Style
+
+For programming in Java we use [google style](https://google.github.io/styleguide/javaguide.html) 
+
+## Javadoc
```

Comment:
This section is possibly redundant, as Google Style Guide includes a Javadoc section